src/utils/logger.py

```diff
@@ -0,0 +1,18 @@
+import logging
+
+
+def get_logger() -> logging.Logger:
+    """
+    Get the logger for the application and set it up
+    :return: logger
+    """
+    logger = logging.getLogger("lynx")

```

Comment:
use `__name__` instead of "lynx".
It seems to be a good python practice.
https://www.palkeo.com/en/blog/python-logging.html